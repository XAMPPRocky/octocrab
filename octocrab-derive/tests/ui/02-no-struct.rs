#![allow(dead_code)]

#[derive(octocrab_derive::Builder)]
enum Test {}

fn main() {}
