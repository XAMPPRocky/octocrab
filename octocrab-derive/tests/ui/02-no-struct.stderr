error: only structs supported
 --> $DIR/02-no-struct.rs:4:1
  |
4 | enum Test {}
  | ^^^^^^^^^^^^
